inc/saifdb.h

```diff
@@ -57,9 +70,15 @@ typedef struct _sai_fdb_entry_t
     /** Mac address */
     sai_mac_t mac_address;
 
-    /** Vlan ID */
+    /** Bridge type */
+    sai_fdb_entry_bridge_type_t bridge_type;
```

Comment:
You have removed vlan from the fdb attribute, vlan is not part of the bridge object then how does the existing vlan object work.